src/components/dashboard/SendTokens/BitcoinLikeBockchain/Form.jsx

```diff
@@ -83,7 +81,6 @@ export function mapStateToProps (state, ownProps) {
   }
 }
 
-@reduxForm({ form: FORM_SEND_TOKENS, validate })
```

Comment:
?! Why?

Reply:
I don’t understand the question. Please, can you explain what you asked about?